pkg/asset/manifests/kubevirt/cloudproviderconfig.go

```diff
@@ -0,0 +1,34 @@
+package kubevirt
+
+import (
+	"bytes"
+	"encoding/json"
+)
+
+// CloudProviderConfig is the kubevirt cloud provider config
+type CloudProviderConfig struct {
+	// The namespace in the infra cluster that the cluster resources are created in
+	Namespace string
+	InfraID   string
+}
+
+type config struct {
```

Comment:
Why do we need two separate structs? Can't we put the json tags on the fields in `CloudProviderConfig` and marshal that?

Reply:
Fixed